plugin.video.hbogoeu/hbogolib/uuid_device.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
+# based on https://github.com/CastagnaIT/plugin.video.netflix/blob/master/resources/lib/common/uuid_device.py
+
+from __future__ import absolute_import, division, unicode_literals
+
+import sys
+
+from kodi_six import xbmc, xbmcaddon
+
+__CRYPT_KEY__ = None
+
+
+def debug(msg):
+    xbmc.log("[" + str(xbmcaddon.Addon().getAddonInfo('id')) + "]" + msg, xbmc.LOGDEBUG)
+
+
+def get_system_platform():
+    platform = "unknown"
+    if xbmc.getCondVisibility('system.platform.linux') and not xbmc.getCondVisibility('system.platform.android'):
+        platform = "linux"
+    elif xbmc.getCondVisibility('system.platform.linux') and xbmc.getCondVisibility('system.platform.android'):
+        platform = "android"
+    elif xbmc.getCondVisibility('system.platform.xbox'):
```

Comment:
this is not available in kodi. If you mean the uwp platform use `system.platform.uwp`. See: https://codedocs.xyz/xbmc/xbmc/modules__infolabels_boolean_conditions.html#modules__infolabels_boolean_conditions_System

Reply:
Fixes tnx